docs/BEST_PRACTICES.md

```diff
@@ -1,3 +1,69 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Order creation best practices](#order-creation-best-practices)
+    - [AFTER a successful payment](#after-a-successful-payment)
+      - [Limitations & Solutions](#limitations--solutions)
+    - [BEFORE a successful payment](#before-a-successful-payment)
+      - [Limitations](#limitations)
+      - [Conclusion](#conclusion)
+- [Payment related best practices](#payment-related-best-practices)
+- [Order/Cart related best practices](#ordercart-related-best-practices)
+- [Deployment best practices](#deployment-best-practices)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Order creation best practices
+#### AFTER a successful payment
+##### Limitations & Solutions
+- It is possible that shop was not reachable for network issues. In that case there will be a successful payment but no order.
```

Comment:
5cfa6c0

Reply:
I created a suggestion so you do not have to create a separate commit and link since just a click on the button above "Commit suggestion" would resolve the issue and commit it for you ;)